dir.props

```diff
@@ -312,7 +263,11 @@
     <IntermediateOutputRootPath Condition="'$(IntermediateOutputRootPath)' == ''">$(BaseIntermediateOutputPath)$(OSPlatformConfig)/</IntermediateOutputRootPath>
     <IntermediateOutputPath Condition="'$(IntermediateOutputPath)' == ''">$(IntermediateOutputRootPath)$(MSBuildProjectName)/$(TargetOutputRelPath)</IntermediateOutputPath>
 
-    <RuntimeDir Condition="'$(RuntimeDir)'==''">$(BinDir)netcoreapp\Runtime</RuntimeDir>
+    <RuntimePath Condition="'$(RuntimePath)' == ''">$(BinDir)runtime/$(BuildConfiguration)/</RuntimePath>
+    <RefRootPath>$(BinDir)ref/</RefRootPath>
```

Comment:
Is there a plan to ensure buildconfigurstion always contains all of the segments? Ie.  If someone specifies netcoreapp-windows_nt,  is that different than specifying netcoreapp-windows_nt-debug-x64,  or is it the same but produces a different location?